include/uv-win.h

```diff
@@ -647,3 +647,17 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
 #ifndef X_OK
 #define X_OK 1
 #endif
+
+#if _MSC_VER >= 1900
+#undef stdin
```

Comment:
with vs2015 update 3 or later version,

application compile failed.
unresolved external symbol __imp____iob_func referenced in ...